manager/manager.go

```diff
@@ -477,13 +487,24 @@ func (m *Manager) ManageVolume(volumeID string) error {
 				}
 
 				if meta.NextIssuanceTime == nil || m.clock.Now().After(*meta.NextIssuanceTime) {
-					log.Info("Triggering new issuance")
-					if err := m.issue(volumeID); err != nil {
-						log.Error(err, "Failed to issue certificate")
-						// retry the request in 1 second time
-						// TODO: exponentially back-off
-						continue
-					}
+					wait.ExponentialBackoffWithContext(ctx, wait.Backoff{
```

Comment:
Change seems good to me and happy to merge.

The second here is not great. What is stopping us constructing a timer at every loop, which takes `*meta.NextIssuanceTime` as input? Can this value change outside of `m.issue`?

Reply:
Agreed - I'll be following up with more PRs to adjust this timer to not just be a fixed 1s, as we can be more intelligent than that :)